### ISSUE TITLE
finetuning problem

### ISSUE THREAD

**Issue by devloper13** [open-mmlab/mmaction2#1376]
I am trying to fintuning i3d on a custom dataset (RawframeDataset format) over kinetics pretrianed model. But it doesn't seem to learn anything right from the first epoch. The error does not decrease. I have used 0.005 as the learning rate as mentioned in the tutorials. What could be the problem?

Also how do we configure the network in such a way that only last few layers are being finetuned and not the entire network?

**Comment by dreamerlin** [comment]
1. How about setting lr = 0.001 with 8 batch size per GPU like [, since ](https://github.com/open-mmlab/mmaction2/blob/master/configs/recognition/tsn/tsn_r50_320p_1x1x8_50e_activitynet_video_rgb.py#L82)
2. You can change `frozen_stages` (https://github.com/open-mmlab/mmaction2/blob/master/mmaction/models/backbones/resnet.py#L344) for last few layers in backbone.

**Comment by devloper13** [comment]
Thanks for the suggestion. setting higher batch size worked. Had given just 1 before.
